PCbuild/_socket.vcxproj

```diff
@@ -60,6 +60,9 @@
     <_ProjectFileVersion>10.0.30319.1</_ProjectFileVersion>
   </PropertyGroup>
   <ItemDefinitionGroup>
+    <ClCompile>
+      <PreprocessorDefinitions>_WINSOCK_DEPRECATED_NO_WARNINGS;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+    </ClCompile>
```

Comment:
Yeah, this is a tough one to fix, but it eventually needs to be fixed.

I deliberately left the warnings there as motivation - we should not suppress them.